app/pb_kits/playbook/pb_circle_icon_button/_circle_icon_button.scss

```diff
@@ -0,0 +1,51 @@
+@import "../tokens/border_radius";
+@import "../tokens/colors";
+@import "../tokens/typography";
+@import "../pb_icon/icon";
+@import "../pb_button/button_mixins";
+
+@mixin pb_circle_icon_button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  object-fit: cover;
+  overflow: hidden;
+  position: relative;
+  width: 40px;
+  height: 40px;
+  border-radius: 20px;
```

Comment:
We will want to use the appropriate border-radius token from the import above ^^